server/container_create_linux.go

```diff
@@ -867,6 +872,7 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrfactory.Cont
 	ociContainer.SetSpec(specgen.Config)
 	ociContainer.SetMountPoint(mountPoint)
 	ociContainer.SetSeccompProfilePath(seccompRef)
+	ociContainer.SetRuntimePathForPlatform(runtimePath)
```

Comment:
Should this only be set when not empty?

Reply:
That's fine bc later on we will check and assign the value: https://github.com/cri-o/cri-o/pull/7180/files#diff-bcc340e07299e056e27ad412ee0817b6f9d8f7350834c1a4214cf1e253f66264R766